itermocil 0.1.6

TomAnthony opened the pull request:
[iTermocil](https://github.com/TomAnthony/itermocil) allows you setup pre-configured layouts of windows and panes in iTerm2, having each open in a specified directory and execute specified commands. It is compatible with teamocil files designed for tmux sessions.

I did previously submit this ~7 weeks ago (https://github.com/Homebrew/homebrew/pull/42549) right after I launched it but it hadn't yet proved whether it would be popular. Since then it has gained a decent amount of popularity and continues to do so, and it would be great to get it into homebrew proper.

dunn commented:
Looks like it's not finding pyyaml:
```
==> /usr/local/Cellar/itermocil/0.1.6/bin/itermocil -h
Traceback (most recent call last):
  File "/usr/local/Cellar/itermocil/0.1.6/libexec/bin/itermocil", line 3, in <module>
    from itermocil import main
  File "/usr/local/Cellar/itermocil/0.1.6/libexec/bin/itermocil.py", line 8, in <module>
    import yaml
ImportError: No module named yaml
Error: itermocil: failed
Failed executing: /usr/local/Cellar/itermocil/0.1.6/bin/itermocil -h
```

TomAnthony commented:
I removed a bit of formula and tested but PyYaml was already linked on my machine so I missed that I needed it. I have fixed and tested again (with PyYaml missing to start with). Thanks.

TomAnthony commented:
@dunn Do I need to open a new PR now this is ready for a revisit? Thanks.

DomT4 commented on Library/Formula/itermocil.rb:
```diff
@@ -0,0 +1,32 @@
+  end
+
+  def install
+    resource("PyYAML").stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
```
You seem to be installing the same resource twice here, in the same place?

DomT4 replied:
Ping?

DomT4 commented:
Closing as dead. Please feel free to resubmit as fresh with the comments above addressed and audit passing, if you'd like to. Thanks for the submission to Homebrew!
